cmd/convert.go

```diff
@@ -97,7 +145,7 @@ func init() {
 	convertCmd.Flags().MarkHidden("deployment")
 
 	// OpenShift only
-	convertCmd.Flags().BoolVar(&ConvertDeploymentConfig, "deployment-config", true, "Generate an OpenShift deploymentconfig object")
+	convertCmd.Flags().BoolVar(&ConvertDeploymentConfig, "deployment-config", false, "Generate an OpenShift deploymentconfig object")
```

Comment:
why are we changing this to false? this should stay as true. we generate deployment configs by default when using the openshift provider.